### PR TITLE
Adding Assignment A02 in branch A02

### PR DESCRIPTION
**Pull request by Ajay07Metamation**

### CODE REVIEW

**Review by rajeevramanTMM — CHANGES_REQUESTED** [review]

File: Training/Program.cs [hunk 1]
```diff
@@ -1,7 +1,18 @@
-﻿namespace Training {
-    internal class Program {
-        static void Main (string[] args) {
-            Console.WriteLine ("Hello, World!");
-        }
+﻿int random = new Random ().Next (1, 101); //Variable to store random number generated
```
[the remark this answers is not in the document]
Comment (Ajay07Metamation) [reply Training/Program.cs:1]: Ok sir. I will change and send it.

File: Training/Program.cs [hunk 2]
```diff
@@ -1,7 +1,18 @@
...
-            Console.WriteLine ("Hello, World!");
-        }
+﻿int random = new Random ().Next (1, 101); //Variable to store random number generated
+Console.Write ("Guess the number between 1 to 100: ");
+for (int tries=0; ;tries++ ) {
+    int guess = ReadInt();
+    if (guess == random) { // check the guess
+        Console.WriteLine ($"Your guess is correct. You took {tries} attempts."); break;
+    } 
+    else if (guess > random)
```
[the remark this answers is not in the document]
Comment (Ajay07Metamation) [reply Training/Program.cs]: Ok sir. I will change and send it.

File: Training/Program.cs [hunk 3]
```diff
@@ -1,7 +1,18 @@
...
+        Console.WriteLine ("Your guess is too high");
+    else
+        Console.WriteLine ("Your guess is too low");
+}
+int ReadInt () { // To get valid input from user and convert to integer
```
[the remark this answers is not in the document]
Comment (Ajay07Metamation) [reply Training/Program.cs:12]: Ok sir. I will change and send it.

File: Training/Program.cs (hunk 2 above)
[the remark this answers is not in the document]
Comment (Ajay07Metamation) [reply Training/Program.cs]: I have done the changes mentioned.

**Pull request closed without merging**